src/integration-test/java/io/aiven/kafka/connect/IntegrationTest.java

```diff
@@ -314,8 +313,13 @@ final void oneFilePerRecordWithPlainValues(final String compression)
 
     @ParameterizedTest
     @ValueSource(strings = {"none", "gzip", "snappy", "zstd"})
-    final void groupByKey(final String compression) throws ExecutionException, InterruptedException, IOException {
-        final Map<String, String> connectorConfig = awsSpecificConfig(basicConnectorConfig(CONNECTOR_NAME));
+    void groupByKey(final String compression, final TestInfo testInfo)
+        throws ExecutionException, InterruptedException, IOException {
+        final var topicName0 = IntegrationBase.topicName(testInfo);
+        final var topicName1 = IntegrationBase.topicName(testInfo) + "_1";
```

Comment:
Oh, here is how you overcome a limitation I mentioned :+1:

Reply:
[Super minor] In the future, we could make it more generic to create all required topics in advance - considering also reusing the Method name for this purpose.